packages/theme/src/cli/flags.ts

```diff
@@ -18,13 +17,6 @@ export const themeFlags = {
   password: Flags.string({
     description: 'Password generated from the Theme Access app.',
```

Comment:
I don't know if you want this documented or not, but should we write it here so we don't get reports of `--password` flag being inconsistent?

```suggestion
    description: 'Password generated from the Theme Access app. Legacy: Supports custom app passwords for `push` and `pull` commands.',
```

Reply:
Spoke about it - we will keep the description as is